ecosystem/typescript/sdk/src/aptos_account.ts

```diff
@@ -30,6 +32,39 @@ export class AptosAccount {
     return new AptosAccount(HexString.ensure(obj.privateKeyHex).toUint8Array(), obj.address);
   }
 
+  static isValidPath = (path: string): boolean => {
+    if (
+      // eslint-disable-next-line prefer-regex-literals
+      !new RegExp('^m\\/44+\'\\/637+\'\\/[0-9]+\'\\/[0-9]+\'\\/[0-9]+\'+$').test(
+        path,
+      )
+    ) {
+      return false;
+    }
+    return !path
+      .split('/')
+      .slice(1)
+      .map((val: string): string => val.replace('\'', ''))
+      // eslint-disable-next-line no-restricted-globals
+      .some(isNaN as any);
+  };
+
+  static fromDerivePath(path: string, mnemonics: string): AptosAccount {
```

Comment:
Also, check if mnemonics is empty and throw meaning full errors, just in case developers forgot to pass in the mnemonics.

Reply:
You still need to check if the mnemonics are empty. Why should we allow that to happen? Throw an error in such case.

Reply:
Add ts-doc for this method